docs/content/dataset_standards.md

```diff
@@ -271,14 +494,16 @@ When creating a Minari dataset with the `DataCollectorV0` wrapper the default gl
 | ----------------------- | ---------- | ----------- |
 | `total_episodes`        | `np.int64` | Number of episodes in the Minari dataset. |
 | `total_steps`           | `np.int64` | Number of steps in the Minari dataset. |
-| `flattened_observation` | `np.bool`  | If the observation space had to be flattened. Usually for `Dictionary` and `Tuple` spaces. |
-| `flattened_action`      | `np.bool`  | If the action space had to be flattened. Usually for `Dictionary` and `Tuple` spaces.|
 | `env_spec`              | `str`      | json string of the Gymnasium environment spec.|
-| `dataset_name`          | `str`      | Name tag of the Minari dataset. |
+| `dataset_id`            | `str`      | Identifier of the Minari dataset. |
 | `code_permalink`        | `str`      | Link to a repository with the code used to generate the dataset.|
 | `author`                | `str`      | Author's name that created the dataset. |
 | `author_email`          | `str`      | Email of the author that created the dataset.|
 | `algorithm_name`        | `str`      | Name of the expert policy used to create the dataset. |
+| `action_space`          | `str`      | Serialized Gymnasium action space describing actions in dataset. |
+| `observation_space`     | `str`      | Serialized Gymnasium observation space describing observations in dataset. |
```

Comment:
I didn’t look I’m too much detail but maybe it’s worth noting how they get serialized? Pickled I’m assuming if it’s like the rest of our stuff, but we may want to look into transforming them into safetensors as pickle objects can be security vulnerabilities (someone can maliciously upload stuff by modifying the source code and running it thru a local install no matter what kinds of precautions we take, if we use pickle)

Reply:
We wrote custom json serialization/de-serialization strategy which doesn't provide a straightforward path to eval() attacks as far as I know. That's not to say there isn't any way someone could figure out how to do an eval attack. I will add a note to the documentation describing the serialization format.